pkg/ipfailover/keepalived/generator.go

```diff
@@ -36,7 +36,7 @@ func getClientConfig(path string) (*kclient.Config, error) {
 	}
 
 	if err := kclient.LoadTLSFiles(config); err != nil {
-		fmt.Errorf("Unable to load certificate info using credentials from %q: %v", path, err)
+		return nil, fmt.Errorf("Unable to load certificate info using credentials from %q: %v", path, err)
```

Comment:
@smarterclayton @ramr PTAL this change - was reverted from a `glog` I believe, what is the correct usage here?